resources/views/shorten_result.blade.php

```diff
@@ -7,9 +7,15 @@
 @section('content')
 <h3>Shortened URL</h3>
 <input type='text' class='result-box form-control' value='{{$short_url}}' />
+<button id="generateQRCode" class='btn btn-primary back-btn'>Generate QR Code</button>
```

Comment:
Can we use `a` tags here for consistency?

Reply:
Done